package.json

```diff
@@ -42,5 +43,7 @@
     "undef": true,
     "es3": true
   },
-  "dependencies": {}
+  "resolutions": {
```

Comment:
`resolutions` is a yarn feature. `overrides` is the npm equivalent and it is only in new enough npm versions.

Reply:
https://www.npmjs.com/package/npm-force-resolutions
Joys of copy paste.

Reply:
It was the preinstall hook doing all the work. It was resisting building.